README.md

```diff
@@ -22,10 +22,16 @@ really care about.
    interface.
 
 
-- - - 
+- - -
 
 ## scikit-learn
 
 1. [Model serve](sklearn/gae_serve) - Serve model with Google App
 	Engine and Cloud Endpoints.
 
+- - -
+
+## Google Cloud Platform
+
+1. [Compute Engine survival training](gce/survival-training/README.md) - A guide to and a
```

Comment:
Compute Engine => Google Compute Engine since thats the product name.

Reply:
Will defer to @thedriftofwords on this one.